graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/ConfigTypesGenerator.java

```diff
@@ -309,4 +273,33 @@ private List<Node> generateApiKeysConfigRecordFields(AuthConfig authConfig) {
         }
         return ballerinaApiKeysConfigRecords;
     }
+
+    /**
+     *  Generates annotations for sensitive fields.
```

Comment:
Remove the extra space before the word `Generates`. A single space is enough.

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/commit/820d69eccf3d162405d31e4797434fb404c1da4e